packages/cli/src/commands/lint.ts

```diff
@@ -18,6 +18,24 @@ import { Totals } from '../types';
 import { blue, gray } from 'colorette';
 import { performance } from 'perf_hooks';
 
+export function mergeLintConfigs(entrypoint: any, config: any) {
+  if (!entrypoint.alias) return config;
+  let mergedLint = config.apis[entrypoint.alias]?.lint || {};
```

Comment:
I'm not totally clear here, but what if the `config` is empty or doesn't exist?

Reply:
@adamaltman that's ok, we always have root lint with default plugins inside - will merge successfully anyway.